test/parser/src/main/resources/sql/supported/dml/select.xml

```diff
@@ -155,4 +155,5 @@
     <sql-case id="select_xmlroot_function" value="SELECT XMLROOT(XMLType('143598'), VERSION '1.0', STANDALONE YES) AS 'XMLROOT' FROM DUAL;" db-types="Oracle" />
     <sql-case id="select_xmlserialize_function" value="SELECT XMLSERIALIZE(DOCUMENT c2 AS BLOB ENCODING 'UTF-8' VERSION 'a' IDENT SIZE = 0 SHOW DEFAULT) FROM b;" db-types="Oracle" />
     <sql-case id="select_from_xmltable_function" value="SELECT warehouse_name warehouse, warehouse2.Water, warehouse2.Rail FROM warehouses, XMLTABLE('/Warehouse' PASSING warehouses.warehouse_spec COLUMNS &quot;Water&quot; varchar2(6) PATH 'WaterAccess',&quot;Rail&quot; varchar2(6) PATH 'RailAccess') warehouse2;" db-types="Oracle" />
+    <sql-case id="select_with_offset" value="select * from t_new_order fetch next 3 row only;" db-types="openGauss" />
```

Comment:
Hi @Qianyi951015, can you see if PostgreSQL also has this problem?

Reply:
I will check now

Reply:
> Hi @Qianyi951015, can you see if PostgreSQL also has this problem?

U r right , PostgreSQL has the same problem, I will fix it soon